lib/std/Build/Step/InstallDir.zig

```diff
@@ -101,6 +104,40 @@ fn make(step: *Step, prog_node: *std.Progress.Node) !void {
 
         switch (entry.kind) {
             .directory => try cwd.makePath(dest_path),
+            .sym_link => {
+                if (!self.options.create_sym_links)
+                    continue;
+
+                var buffer: [fs.MAX_NAME_BYTES]u8 = undefined;
+                const target_path = try entry.dir.readLink(entry.basename, buffer[0..]);
+
+                while (true) {
+                    cwd.symLink(target_path, dest_path, .{}) catch |err| switch (err) {
+                        error.PathAlreadyExists => {
+                            const st = try cwd.statFile(dest_path, .{ .follow_symlinks = false });
```

Comment:
I really not sure if I went too far with this, but I had to add a `StatOptions` to `Dir.statFile()` with a flag to not follow the symlinks so I can identify the type of the current entry. Luckily it was easy thing to do because there were not much places calling this function.